config/rbac/role.yaml

```diff
@@ -189,6 +213,18 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - networking.k8s.io
+  resources:
+  - networkpolicies
```

Comment:
@koncar Not sure if Ketch deals with network policies,  wonder why do we need network policies permissions?

Reply:
removed